pkgs/development/php-packages/grumphp/default.nix

```diff
@@ -1,32 +1,24 @@
-{ mkDerivation, fetchurl, makeWrapper, lib, php }:
+{ fetchFromGitHub, lib, php }:
 
-mkDerivation (finalAttrs: {
+php.buildComposerProject (finalAttrs: {
   pname = "grumphp";
-  version = "1.15.0";
+  version = "2.0.0";
 
-  src = fetchurl {
-    url = "https://github.com/phpro/grumphp/releases/download/v${finalAttrs.version}/grumphp.phar";
-    sha256 = "sha256-EqzJb7DYZb7PnebErLVI/EZLxj0m26cniZlsu1feif0=";
+  src = fetchFromGitHub {
+    owner = "phpro";
+    repo = "grumphp";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-gQs05nrKh1A6N6xmoPayqoqFZdV2zK/2UAg6ZkpDzBs=";
   };
 
-  dontUnpack = true;
+  vendorHash = "sha256-iaLhs1lKo4X2ix6WjtgJDKc2NiPQpOlwrSR8ois6aaM=";
```

Comment:
The attribute `composerLock` is missing.

Context: https://github.com/phpro/grumphp/issues/1095

Possible solution:

```
  composerLock = fetchurl {
    url = "https://github.com/phpro/grumphp-shim/raw/695f2f0be5f2790242c362e8b79d3f92c276c9a1/phar.composer.lock";
    hash = "sha256-XeWDUQUnUEOCtT+wjmR0kXikGHdE336KXglTfzfVVQo=";
  };
```

Reply:
Well, it does build for me from this commit that is pushed. I tested adding the following `installPhase`: `ls; exit 1` and there's a `composer.lock` in there. So I guess they have added it to the release tarball?

Reply:
Not sure how I got it to build once. I'm dropping the changes for that package for now.